src/layouts/Layout.astro

```diff
@@ -84,11 +85,19 @@ const socialImageURL = new URL(
     <meta property="twitter:image" content={socialImageURL} />
 
     <!-- Google Font -->
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:ital,wght@0,400;0,500;0,600;0,700;1,400;1,600&display=swap"
-      rel="stylesheet"
+    <AstroFont
+      config={[
+        {
+          src: [],
+          name: "IBM Plex Mono",
+          googleFontsURL:
+            "https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:ital,wght@0,400;0,500;0,600;0,700;1,400;1,600&display=swap",
+          preload: false,
+          display: "swap",
+          fallback: "sans-serif",
```

Comment:
I'm not very clear about this. So, correct me if I'm wrong. I think fallback should be 'mono' since the font-family of the site is using a mono font, am I right?

Reply:
I've noticed that @rishi-raj-jain posted the same PR on other themes.  He might be more of an evangelist for  Astro-Font (I've been there for jsDelivr).

Might want to fix yourself or just close.